static/includes/apps/AppsDatasets.md

```diff
@@ -0,0 +1,63 @@
+&NewLine;
+
```

Comment:
Just curious, why are we adding this content as a snippet? Where would this content get inserted as a  snippet?
I don't see including this in the individual tutorials.

Reply:
It's an existing snippet I've just improved and renamed to something that makes more sense with the content. It's currently included in the apps index and enterprise apps index. Potentially it would work in the Community and Stable indices too. 

That said, it's now only called in AppsBestPractices.md since that wrapper content was nearly identical in the two indexes. I could potentially remove the child snippets  and embed that directly in the parent, if they're not being called elsewhere, but there might be something to be said for keeping it more modular in case they do apply in other contexts?